packages/expo-updates/cli/utils/args.ts

```diff
@@ -0,0 +1,42 @@
+// Common utilities for interacting with `args` library.
```

Comment:
@EvanBacon - do you think it's worth factoring these and logging utils and modifyConfig out into a library somewhere in this repo? (they're copypasta'd from your CLI)